internal/expanders/gatekeeper.go

```diff
@@ -68,7 +68,8 @@ func (g GatekeeperPolicyExpander) Expand(
 				"severity":          severity,
 				"object-templates": []map[string]interface{}{
 					{
-						"complianceType": "musthave",
+						"complianceType":         "musthave",
+						"metadataComplianceType": "",
```

Comment:
@dhaiducek  @mprahl Need to confirm metadataComplianceType in gatekeeper expander

Reply:
@ChunxiAlexLuo I would leave this out in the expanders.